rollup-common.js

```diff
@@ -453,6 +454,13 @@ export function litProdConfig({
                     'const ENABLE_SHADYDOM_NOPATCH = false',
                 },
               }),
+              inject({
+                HTMLElement: ['@lit-labs/ssr-dom-shim', 'HTMLElement'],
+                customElements: ['@lit-labs/ssr-dom-shim', 'customElements'],
+                include: [
+                  '**/packages/reactive-element/development/reactive-element.js',
+                ],
+              }),
```

Comment:
Inject plugin is a fantastic solution to providing global only for the node builds! We do need this also for the dev node build below, ~line 507.

Reply:
Done